doc/howto/network_ipam.md

```diff
@@ -16,27 +16,17 @@ By default, this command shows the IPAM information for the `default` project. Y
 The resulting output will look something like this:
 
 ```
-{
-  "cidr_addresses": [
-    "10.0.0.1/32",
-    "::ffff:a00:1/128"
-  ],
-  "used_by": "/1.0/networks/lxdbr0",
-  "type": "network",
-  "nat": false,
-  "hwaddr": ""
-}
-
-{
-  "cidr_addresses": [
-    "::ffff:a00:79/128",
-    "10.0.0.121/32"
-  ],
-  "used_by": "/1.0/instances/u1",
-  "type": "instance",
-  "nat": false,
-  "hwaddr": "00:16:3e:13:09:9e"
-}
++----------------------+-------------------------------------------+----------+------+-------------------+
+|       USED BY        |                  ADDRESS                  |   TYPE   | NAT  | HARDWARE ADDRESS  |
++----------------------+-------------------------------------------+----------+------+-------------------+
+| /1.0/networks/lxdbr0 | 10.6.105.1/24                             | network  | true |                   |
++----------------------+-------------------------------------------+----------+------+-------------------+
+| /1.0/networks/lxdbr0 | fd42:3cce:990:a1fd::1/64                  | network  | true |                   |
++----------------------+-------------------------------------------+----------+------+-------------------+
+| /1.0/instances/u1    | fd42:3cce:990:a1fd:216:3eff:fe04:f095/128 | instance | true | 00:16:3e:04:f0:95 |
++----------------------+-------------------------------------------+----------+------+-------------------+
+| /1.0/instances/u1    | 10.6.105.160/32                           | instance | true | 00:16:3e:04:f0:95 |
```

Comment:
Would it make sense to use documentation-specific IP addresses?
https://en.wikipedia.org/wiki/Reserved_IP_addresses

Reply:
Yes I think it would (we always should). I'm going to merge this now as I need to get the release tag done. But @gabrielmougard can do a follow up one with this good improvement.

Reply:
yes

Reply:
@ru-fu here it is https://github.com/canonical/lxd/pull/12060